content/docs/2.4/scalers/kubernetes-workload.md

```diff
@@ -0,0 +1,43 @@
++++
+title = "Kubernetes Workload"
+layout = "scaler"
+availability = "v2.4+"
+maintainer = "Community"
+description = "Scale applications based on the amount of pods which matches the given selectors."
+go_file = "kubernetes_workload_scaler"
++++
+
+### Trigger Specification
+
+```yaml
+triggers:
+- type: kubernetes-workload
```

Comment:
What is the authentication story here? It uses the KEDA identity I presume? Maybe good to document that as well?

Reply:
You're right, I will write it explicitly

Reply:
done!